chevah/compat
# Fix docker tests.

Scope
=====

This is a small PR to fix the docker tests on centos.

Changes
=======

set umask like chevah/server

use /usr/bin/curl for a reference binary.

The bin/cp was a small file on centos and could be confused with a symlink.

I have also updated the compat brink.sh and python version.

----------

On alpine, PAM usage is crashing big times, with segfault.
For now, I have just disable it as I don't plan to use PAM on alpine.


How to try and test the changes
===============================

reviewers: @dumol

Check that changes make sense.



Scope
=====

This is a small PR to fix the docker tests on centos and alpine.

The centos issue was minor.
The alpine is big as trying to call PAM will segfault. For now we just skip PAM on Alpine.
